solutions/task_1.py

```diff
@@ -0,0 +1,62 @@
+class Stars:
+    def __init__(self, star_name=None):
+        if star_name is not None:
+            self.name = star_name
+        else:
+            self.name = "Acrux"
+
+    def __eq__(self, other):
+        return self.name == other.name
+
+    def __hash__(self):
+        return hash(self.name)
+
+
+def func_1():
+    return Stars
+
+
+def func_2():
+    return func_1
+
+
+if __name__ == '__main__':
+    foo = [
+        Stars,
+        {"Sirius": [Stars(), Stars]},
+        {"Arcturus": [[{"Vega": Stars}]],
+         "Capella": {"Rigel": {"Procyon": func_1,
```

Comment:
pep8

Reply:
Already done

Reply:
When I said "Make it in single line" I mean one element of the list per line.
Anyway I fixed it by myself, see comments below.